packages/grid/_modules_/grid/models/api/gridEditRowApi.ts

```diff
@@ -4,68 +4,75 @@ import { GridRowId } from '../gridRows';
-   * @param GridEditRowsModel
+   * Set sthe edit rows model of the grid.
+   * @param {GridEditRowsModel} model The new edit rows model.
    */
   setEditRowsModel: (model: GridEditRowsModel) => void;
   /**
-   * Get the edit rows model of the grid.
-   * @returns GridEditRowsModel
+   * Gets the edit rows model of the grid.
+   * @returns {GridEditRowsModel} The edit rows model.
    */
   getEditRowsModel: () => GridEditRowsModel;
   /**
-   * Set the cellMode of a cell.
-   * @param GridRowId
-   * @param string
-   * @param 'edit' | 'view'
+   * Sets the mode of a cell.
+   * @param {GridRowId} id The id of the row.
+   * @param {string} field The field to change the mode.
+   * @param {GridCellMode} mode Can be: `"edit"`, `"view"`.
    */
   setCellMode: (id: GridRowId, field: string, mode: GridCellMode) => void;
   /**
-   * Get the cellMode of a cell.
-   * @param GridRowId
-   * @param string
-   * @returns 'edit' | 'view'
+   * Gets the mode of a cell.
+   * @param {GridRowId} id The id of the row.
+   * @param {string} field The field to get the mode.
+   * @returns Returns `"edit"` or `"view"`.
    */
   getCellMode: (id: GridRowId, field: string) => GridCellMode;
   /**
-   * Returns true if the cell is editable.
-   * @param params
+   * Controls if a cell is editable.
+   * @param {GridCellParams} params The cell params.
+   * @returns {boolean} A boolean value determining if the cell is editable.
    */
   isCellEditable: (params: GridCellParams) => boolean;
   /**
-   * Set the edit cell input props.
-   * @param rowId
-   * @param update
+   * Sets the input props of the edit cell.
+   * @param {GridEditCellPropsParams} params The params to set.
    */
   setEditCellProps: (params: GridEditCellPropsParams) => void;
   /**
-   * Get the edit cell input props.
-   * @param rowId
-   * @param field
+   * Gets the input props for the edit cell of a given `rowId` and `field`.
+   * @param {GridRowId} rowId The id of the row.
+   * @param {string} field The field name.
+   * @returns {GridEditCellProps} The props for the edit cell.
    */
   getEditCellProps: (rowId: GridRowId, field: string) => GridEditCellProps;
   /**
-   * Get the edit cell input props params passed in handler.
-   * @param rowId
-   * @param field
+   * Gets the params to be passed when calling `setEditCellProps`.
+   * @param {GridRowId} rowId The id of the row.
+   * @param {string} field The field name.
+   * @returns {GridEditCellPropsParams} The params.
    */
   getEditCellPropsParams: (rowId: GridRowId, field: string) => GridEditCellPropsParams;
   /**
    * Get the edit cell value params.
    * @param rowId
    * @param field
+   * @ignore - do not document
    */
   getEditCellValueParams: (rowId: GridRowId, field: string) => GridEditCellValueParams;
   /**
    * Commit the cell value changes to update the cell value.
    * @param update
+   * @ignore - do not document
    */
   commitCellChange: (params: GridEditCellPropsParams) => void;
```

Comment:
I don't see a rationale to expose this method in the API. Although used in [this](https://material-ui.com/components/data-grid/editing/#edit-using-external-button) demo, the same thing can be done with `apiRef.current.setCellValue`.

Reply:
@m4theushw "commit" means that the value is stored and will be taken part of the ctrl + z, ctrl + alt + z history feature. set value means that the value is only locally updated without updating the underlying rows. I would personally do the opposite, kill `setCellValue`.

Reply:
If there would be a history feature then makes sense to have the "commit". I added the ignore to `setCellValue` for now.